docs/src/pages/getting-started/templates/blog/Markdown.tsx

```diff
@@ -1,20 +1,23 @@
 import * as React from 'react';
 import ReactMarkdown from 'markdown-to-jsx';
-import {
-  Theme,
```

Comment:
I don't know why these type imports are problematic at this particular situation.